### ISSUE TITLE
global: author and subject identifiers REST API support

### ISSUE THREAD

**Issue by lnielsen** [zenodo/zenodo#217]
- [x] Record data model update 
- [x] REST API support for authors
- [x] GND control number validation: http://en.wikipedia.org/wiki/Wikipedia:Authority_control  
- [ ] Multiple or single identifier per scheme?  
~~Support for LCCN, SELIBR, VIAF, ISNI?~~

**Comment by lnielsen** [comment]
ef3707dcf676113a8c85b438ca354b520beb91f4

**Comment by lnielsen** [comment]
MARC21 example:

```xml
<datafield tag="100" ind1=" " ind2=" ">
<subfield code="a">Familyname, Given names</subfield>
<subfield code="0">(orcid)0000-0003-1234-1234</subfield>
<subfield code="0">(gnd)...</subfield>
</datafield>
```

Identifier codes:
http://www.loc.gov/standards/sourcelist/standard-identifier.html

**Comment by bouzlibop** [comment]
```xml
<nameIdentifier schemeURI="http://orcid.org" nameIdentifierScheme="ORCID">http://orcid.org/0000-0001-5000-0007</nameIdentifier>
```

**Comment by lnielsen** [comment]
See also #106

**Comment by lnielsen** [comment]
- [ ] Add new field ``subjects`` in ``zenodo.cfg``. List of objects, object properties (term, id, scheme)
- [ ] Store data in ``650 1_ $a<TERM>$0(<SCHEME>)<IDENTIFIER>``.
- [ ] REST API support - docs need to communicate the new field as a "preview period".
- [ ] Deposit form: (show term, id, but hide scheme - see related identifiers for example)
- [ ] Record UI: display subjects in the metadata box.
- [ ] Update DataCite XML export

**Comment by bouzlibop** [comment]
First iteration on subjects addition:
![fisrt_iter](https://cloud.githubusercontent.com/assets/8625570/8126355/1766bf82-10ee-11e5-941f-3ac95d970e82.png)

**Comment by bouzlibop** [comment]
@lnielsen, this is an example from [DataCite page](https://schema.datacite.org/meta/kernel-3/example/datacite-example-full-v3.1.xml):
```
<subjects>
    <subject xml:lang="en-us" schemeURI="http://dewey.info/" subjectScheme="dewey">000 computer science</subject>
</subjects>
```

how it should look like in our case - `650 1_ $a<TERM>$0(<SCHEME>)<IDENTIFIER>`?
something like that:
```
<subjects>
    <subject subjectScheme="<SCHEME>"><IDENTIFIER> <TERM></subject>
</subjects>
```
?

**Comment by lnielsen** [comment]
Great progress :-)

**Subjects in DataCite:**

Ouch, thought there was a name identifier in the subject...ups. I would say:
```xml
<subjects>
    <subject subjectScheme="SCHEME">IDENTIFIER</subject>
</subjects>
```

**Comment by lnielsen** [comment]
**Clash with 6501_ with 65017**

We should be good, since we don't use ind2=7.

**Comment by lnielsen** [comment]
Mixing 650 with 653 in DataCite:Subjects is fine.

**Comment by lnielsen** [comment]
Topical term -> Term
Authority record control number -> Term identifier

**Comment by lnielsen** [comment]
**Metadata box**
Not sure if we should mix Keywords/Subjects into one. Leave as is for now, until we take a decision.

**Comment by lnielsen** [comment]
I need to provide:
1) Preview period warning.
2) Subject initial description

**Comment by lnielsen** [comment]
Closed via 697b61422841225ae60106dadfe04445587bafdc
